go.mod

```diff
@@ -13,7 +13,7 @@ require (
 	github.com/pborman/uuid v1.2.1
 	github.com/robfig/cron v1.2.0
 	github.com/stretchr/testify v1.10.0
-	go.temporal.io/api v1.43.2
```

Comment:
This will need to be a released version of the API to release this change.

Reply:
I'd argue we need a released version of the server and a test proving this works to even merge